automl/cloud-client/get_operation_status_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud import automl
+import pytest
+
+import get_operation_status
+
+PROJECT_ID = os.environ["AUTOML_PROJECT_ID"]
+
+
+@pytest.fixture(scope="function")
+def get_operation_id():
```

Comment:
nit: Usually fixture names are just the name of the thing
```suggestion
def operation_id():
```